### PR TITLE
Seven story 7s

### PR DESCRIPTION
**Pull request by chchitti**

### CODE REVIEW

File: assets/stylesheets/generics/colors.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
   --alto: #dadada;
   --gray: #929292;
   --mariner: #2f81cd;
-
+  --mine-shaft: #333333;
```
Comment (nandap4790) [inline assets/stylesheets/generics/colors.scss]: we need not add these colors for now, let it be the default colors

File: src/components/atoms/StoryCard/index.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import { get } from "lodash";
+import { Link } from "@quintype/components";
+import Contributor from "../../atoms/Contributor";
+import ResponsiveImageWithFallback from "../ResponsiveImageWithFallback";
+import Icon from "../Icon";
+
+import { getStoryData, generateImageSources } from "../../utils/utils";
+// import { isPremium } from "../../../../isomorphic/data/story";
+
+import "./styles.m.css";
```
Comment (nandap4790) [inline src/components/atoms/StoryCard/index.js]: can this file be renamed to `storyCard.m.css` ?

File: src/components/atoms/StoryCard/index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        {story.engagement && story.engagement.total && (
+          <span styleName="engagement">
+            <Icon type="speaker" styleName="speaker-icon" />
+            {story.engagement.total}
+          </span>
+        )}
+      </ResponsiveImageWithFallback>
+      <div styleName="content">
+        {contributor && <Contributor name={contributor["name"]} type={contributorRole} styleName="contributor" />}
+        <h3 styleName={`headline ${hasTruncatedHeadline ? "truncated" : ""} `}>{storyData.headline}</h3>
```
Comment (nandap4790) [inline src/components/atoms/StoryCard/index.js]: There is a headline component, which you can make use of

File: src/components/atoms/StoryCardFullWidth/index.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+import { Link } from "@quintype/components";
+import Contributor from "../../atoms/Contributor";
+import ResponsiveImageWithFallback from "../ResponsiveImageWithFallback";
+import { getStoryData, generateImageSources } from "../../utils/utils";
+import Icon from "../Icon";
+import { getMonthDayString } from "../../utils/time-format";
+// import { isPremium } from "../../../../isomorphic/data/story";
+
+import "./styles.m.css";
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/index.js]: this should be renamed

File: src/components/atoms/StoryCardFullWidth/index.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+            </span>
+          )}
+        </ResponsiveImageWithFallback>
+        <div styleName="content">
+          {contributor && (
+            <Contributor
+              name={contributor["name"]}
+              type={contributorRole}
+              iconColor="#929292"
+              styleName="contributor"
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/index.js:70]: this i think should be the className

File: src/components/atoms/StoryCardFullWidth/index.js [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+            />
+          )}
+          <h3 styleName="headline">{storyData.headline}</h3>
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/index.js]: Headline component

File: src/components/rows/SevenStory7s/index.js [hunk 7]
```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CollectionLink from "../../utils/generate-collection-link";
+import StoryCardFullWidth from "../../atoms/StoryCardFullWidth";
+import StoryCard from "../../atoms/StoryCard";
+import Icon from "../../atoms/Icon";
+import "./styles.m.css";
```
Comment (nandap4790) [inline src/components/rows/SevenStory7s/index.js]: rename

File: src/components/rows/SevenStory7s/index.js [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+export const SevenStory7s = ({ collection, Icon, BgPattern = "bg-diamond-dark" }) => {
+  const { name, items } = collection;
+  const MaxNumberOfThumbnailStories = 6;
+  return (
+    <div styleName={`seven-story-7s ${BgPattern}`}>
+      <div className="container">
+        <CollectionLink collection={collection}>
+          <h2 styleName="title"> {Icon ? <Icon /> : name}</h2>
```
Comment (nandap4790) [inline src/components/rows/SevenStory7s/index.js]: Headline component

**Review by nandap4790 — CHANGES_REQUESTED** [review]

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 5 above)
Comment (chchitti) [reply src/components/atoms/StoryCardFullWidth/index.js:70]: I think this should be styleName. In storyCardFullWidth component I am adding margin property that is local for to that component.

File: src/components/rows/SevenStory7s/index.js (hunk 8 above)
Comment (chchitti) [reply src/components/rows/SevenStory7s/index.js]: Done

File: src/components/rows/SevenStory7s/index.js (hunk 7 above)
Comment (chchitti) [reply src/components/rows/SevenStory7s/index.js]: Done

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 6 above)
Comment (chchitti) [reply src/components/atoms/StoryCardFullWidth/index.js]: Done

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 4 above)
Comment (chchitti) [reply src/components/atoms/StoryCardFullWidth/index.js]: Done

File: src/components/atoms/StoryCard/index.js (hunk 3 above)
Comment (chchitti) [reply src/components/atoms/StoryCard/index.js]: Done

File: src/components/atoms/StoryCard/index.js (hunk 2 above)
Comment (chchitti) [reply src/components/atoms/StoryCard/index.js]: Done

File: assets/stylesheets/generics/colors.scss (hunk 1 above)
Comment (chchitti) [reply assets/stylesheets/generics/colors.scss]: Ok.. Actually there is no good color to suit this purpose. Suggest we keep this one

Comment (nandap4790) [reply assets/stylesheets/generics/colors.scss]: that, the designers will take a call, they have their own set of colors

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 5 above)
Comment (nandap4790) [reply src/components/atoms/StoryCardFullWidth/index.js:70]: can be discussed when you are in

Comment (nandap4790) [reply src/components/atoms/StoryCardFullWidth/index.js:70]: let me think of it

**Pull request merged**